.github/workflows/e2e-browser.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-node@v4
         with:
-          node-version: 20.x
+          node-version: 22.x
```

Comment:
Here, the node version is only ever used to build, it's irrelevant to the tests. I suggest we change this to `latest`, to avoid the churn.

Reply:
See #1079